lib/livebook/runtime.ex

```diff
@@ -115,7 +115,10 @@ defprotocol Livebook.Runtime do
           code: String.t()
         }
 
-  @type node_memory :: %{
+  @typedoc """
+  The runtime memory usage for each type in bytes
+  """
+  @type runtime_memory :: %{
```

Comment:
Also document that the runtime may periodically send messages of type `{:memory_usage, runtime_memory()}`. We don't have `@moduledoc` here, but there's a comment imitating that.